### ISSUE TITLE
16$: Fix warning from 'apt update'

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#149]
Running `apt update` emits the following notice:

    N: Skipping acquire of configured file 'main/binary-i386/Packages' as repository 'https://c.quick-lint-js.com/debian experimental InRelease' doesn't support architecture 'i386'

Figure out why this notice appears and shut it up.

**Comment by jgil-r** [comment]
I claim this for-hire task. I expect payment after I complete this task. I will email the quick-lint-js team if I am assigned this task.

**Comment by jgil-r** [comment]
Scratch the for hire pay. I think you just gotta add `[arch=amd64]` to quick lint source list.

So it should be
```bash
printf '\n# https://quick-lint-js.com/install/#debian\ndeb [arch=amd64] https://c.quick-lint-js.com/debian experimental main\n' | sudo tee -a /etc/apt/sources.list.d/quick-lint-js.list
```

I think right now its trying to install packages for an unsupported architecture, so it throws that warning. I can do a pr first thing in the morning.
